duties/protocol/request.py

```diff
@@ -90,11 +91,13 @@ def __send_request(
                         url=f"{ARGUMENTS.beacon_node}{endpoint}",
                         data=calldata,
                         timeout=program.REQUEST_TIMEOUT,
+                        headers=program.REQUEST_HEADER,
                     )
                 case CalldataType.PARAMETERS:
+                    payload = urlencode({"id": calldata}, safe=",")
```

Comment:
this should probably be called `params` to make it a bit clearer

Reply:
that's even better 👍

Reply:
Then I'm looking forward that you close the conversation and approve the PR :smiley: . Will merge today in the evening (CET).

Reply:
I am not able to resolve the conversation, but reviewed again and re-approved it 👍